go/service/main.go

```diff
@@ -275,8 +274,7 @@ func (d *Service) ListenLoop(l net.Listener) (err error) {
 		var c net.Conn
 		if c, err = l.Accept(); err != nil {
 
-			// net.errClosing isn't exported, so do this.. UGLY!
-			if strings.HasSuffix(err.Error(), "use of closed network connection") {
+			if libkb.IsSocketClosedError(err) {
```

Comment:
I am not positive, I just made Windows do what 'nix was doing here for the test to pass. Maybe because closing the pipe is normal?